pkg/nuclide-terminal/lib/terminal-view.js

```diff
@@ -53,6 +53,7 @@ const PRESERVED_COMMANDS_CONFIG = 'nuclide-terminal.preservedCommands';
 const SCROLLBACK_CONFIG = 'nuclide-terminal.scrollback';
 const CURSOR_STYLE_CONFIG = 'nuclide-terminal.cursorStyle';
 const CURSOR_BLINK_CONFIG = 'nuclide-terminal.cursorBlink';
+const FONT_FAMILY_CONFIG = 'nuclide-terminal.fontFamily';
```

Comment:
while we're at it, would you also be willing add terminal-specific font size and line height settings?